### PR TITLE
Minor changes in scripts: extract-sentences.py, ngram-count-patterns-maxent2.py

### PR DESCRIPTION
**Pull request by vivekvardhanadepu**
- updated the number of args in `extract-sentences.py`
- updated some parts of `ngram-count-patterns-maxent2.py` to python3

### CODE REVIEW

**Comment by TinoDidriksen** [comment]
Erk. Your email address is unfortunately private atm. We need it in the log for correct copyright annotation, unless you assign copyright to Apertium.

File: scripts/ngram-count-patterns-maxent2.py [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
-import sys, codecs, copy, commands;
+import sys, codecs, copy;
 import common
-sys.stdin  = codecs.getreader('utf-8')(sys.stdin);
-sys.stdout = codecs.getwriter('utf-8')(sys.stdout);
-sys.stderr = codecs.getwriter('utf-8')(sys.stderr);
+# sys.stdin  = codecs.getreader('utf-8')(sys.stdin);
+# sys.stdout = codecs.getwriter('utf-8')(sys.stdout);
+# sys.stderr = codecs.getwriter('utf-8')(sys.stderr);
```
Comment (unhammer) [inline scripts/ngram-count-patterns-maxent2.py:9]: just delete instead of commenting out

File: scripts/ngram-count-patterns-maxent2.py [hunk 2]
```diff
@@ -178,10 +178,11 @@ def wrap (x):
 						meoutcomes[slword][event_counter] = '';
 					#}
 					for ni in ngrams[slword]: #{
-						if ni not in features: #{
+						if ni not in features: #and ni != "": {
```
Comment (unhammer) [inline scripts/ngram-count-patterns-maxent2.py:181]: what happens when ni=="" here?

File: scripts/ngram-count-patterns-maxent2.py (hunk 1 above)
Comment (vivekvardhanadepu) [reply scripts/ngram-count-patterns-maxent2.py:9]: sure!!

File: scripts/ngram-count-patterns-maxent2.py (hunk 2 above)
Comment (vivekvardhanadepu) [reply scripts/ngram-count-patterns-maxent2.py:181]: oops!! I have to remove that. It was sort of workaround for some error which I am no longer getting :grin: :grin:

**Pull request merged**